pulp-glue-ostree/pulp_glue/ostree/context.py

```diff
@@ -73,14 +73,12 @@ class PulpOstreeRemoteContext(PulpRemoteContext):
 
```

Comment:
You can delete all the `_ID` things above and just keep the `ID_PREFIX`.
Also i would move the version requirement here in to `NEEDS_PLUGINS`.